packages/zoe/src/contracts/loan/liquidate.js

```diff
@@ -0,0 +1,59 @@
+// @ts-check
+import '../../../exported';
+
+import { E } from '@agoric/eventual-send';
+
+import { depositToSeat, withdrawFromSeat } from '../../contractSupport';
+
+/**
+ * This function is triggered by the priceOracle when the value of the
+ * collateral is below the mmr percentage. The function performs the
+ * liquidation and then shuts down the contract. Note that if a
+ * liquidation occurs, the borrower gets nothing and they can take no
+ * further action.
```

Comment:
So when the collateral is only 130% of the borrowed amount, they lose everything, rather than getting 30% back? 
I guess that's incentive to keep it over-collateralized.

Reply:
Yeah, that's the simplest-to-code version. There's a TODO to liquidate partially.